home/tests/test_whatsapp.py

```diff
@@ -53,3 +57,80 @@ def test_create_whatsapp_template_with_buttons(self):
 
         self.assertEquals(request.headers["Authorization"], "Bearer fake-access-token")
         self.assertEquals(request.body, json.dumps(data, indent=4))
+
+    @responses.activate
+    def test_create_whatsapp_template_with_image(self):
+        img_name = "test.jpeg"
+        img_path = f"{settings.TEST_STATIC_ROOT}/{img_name}"
+
+        f = open(img_path, "rb")
+        image = Image(
+            title=img_name,
+            file=ImageFile(f, name=img_path),
+        )
+        mock_file_content = image.file
```

Comment:
I don't think this variable name is true anymore, or needed. (it contains the open file, not the content of that file)

Reply:
Fixed

Reply:
I'm seeing the other changes, but not this one

Reply:
Argh, sorry man, my earlier undo went a bit too far back 😂. It should be there now.  I tried removing the variable and doing the saved_image.file on line 82 where it is added to req_files, but it didn't like that, so I renamed it and renamed f to read_file.